numba/cuda/decorators.py

```diff
@@ -142,6 +142,15 @@ def autojitwrapper(func):
 
 
 def declare_device(name, sig):
+    """
+    Declare the signature of a foreign function. Returns a descriptor that can
+    be used to call the function from a Python kernel.
+
+    :param name: The name of the foreign function.
+    :type name: str
+    :param sig: The Numba signature of the function.
+    :type sig: str
```

Comment:
Should this also accept a `numba.core.typing.templates.Signature` such as that derived from composing a signature with `numba.types` types?

Reply:
It will already accept one, I think I just got carried away with the docs here. I'll just remove this type documentation since a Numba signature can be a `Signature` or a string.

Reply:
Thanks, seems like that's the easiest fix!